LandSandBoat/server
# [sql] Add Endark/Enlight animation and skill type/element

**_I affirm:_**

## What does this pull request do?

Corrects missing animations for Enlight/Endark
Sets skill type so you have non-zero skill for the purposes of checking interrupt
Enables skilling up divine/dark magic for the appropriate spell
![image](https://github.com/LandSandBoat/server/assets/60417494/deb3fa3d-eca2-4d7a-93dc-56c9be221017)

## Steps to test these changes

Use enlight/endark and get skillups on divine/dark, don't get interrupted by mobs and see correct animations.
